pkgs/top-level/python-packages.nix

```diff
@@ -32131,6 +32131,128 @@ EOF
 
   treq = callPackage ../development/python-modules/treq { };
 
+  nss = buildPythonPackage rec {
```

Comment:
also, is it this package? https://pypi.org/project/python-nss/#files
If so, please give it the name PyPI uses, and preferably fetch from there as well.